UI/src/sass/core.scss

```diff
@@ -5,3 +5,17 @@
 .display-pre-wrap {
   white-space: pre-wrap;
 }
+
+.feedback-iframe {
+  display: none;
+}
+
+.js-enabled .feedback-iframe {
+  display: inline;
+  width: 100%;
+  height: 100vh;
+}
+
+.js-enabled .feedback-no-js {
```

Comment:
Should we also add the below so we are not showing the iframe loading screen?
```
 .js-enabled .feedback-iframe {
   display: none;
}
```

Reply:
I may have misunderstood, wouldn't that css hide the iframe completely if js is enabled?

At the moment it's hidden by default:

```
.feedback-iframe {
  display: none;
}
```

Then if js is enabled its is shown, otherwise left as hidden:

```
.js-enabled .feedback-iframe {
  display: inline;
  width: 100%;
  height: 100vh;
}

```

Reply:
make sense.